src/goto-programs/goto_inline.cpp

```diff
@@ -57,7 +57,7 @@ void goto_inline(
       return;
 
     goto_functiont &goto_function=it->second;
-    assert(goto_function.body_available());
+    DATA_INVARIANT(goto_function.body_available(), "body available");
```

Comment:
If possible could the message include a short explanation of why this should be impossible?